Backend/app/rider-platform/rider-app/Main/src/Tools/Event.hs

```diff
@@ -80,6 +88,18 @@ data QuoteEventData = QuoteEventData
     merchantId :: Id Merchant
   }
 
+data ExophoneEventData = ExophoneEventData
+  { vendor :: Maybe Text,
+    callType :: Maybe Text,
+    rideId :: Maybe (Id DRide.Ride),
```

Comment:
Let's make it as entityId instead of rideId. As @hemantmangla told that it can be rcId as well.